test/e2e/workstatus-creation.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+# Copyright 2024 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x # echo so that users can understand what is happening
+set -e # exit on error
+
+:
+: -------------------------------------------------------------------------
+: Build and deploy the addon
+:
+make ko-local-build
+make kind-load-image CLUSTERS="hub cluster1"
```

Comment:
I know it's not part of the PR, but I really prefer we would load images with a command or a script and not through `make`  target

Reply:
This is general for all our Makefiles. So let's close it here, and open as general issue .